### PR TITLE
feat: Support for the Transform section

### PR DESCRIPTION
**Pull request by otaviomacedo**
Fixes https://github.com/cdklabs/decdk/issues/278

### CODE REVIEW

**Review by lejiati — COMMENTED** [review]
I assume this change is only to support transforms declared at template level?

File: src/evaluate/evaluate.ts [hunk 1]
```diff
@@ -65,6 +65,10 @@ export class Evaluator {
       });
       this.context.addReference(new SimpleReference(outputId));
     });
+
+    this.context.template.transform.forEach((t) => {
+      this.context.stack.addTransform(t);
```
Comment (lejiati) [inline src/evaluate/evaluate.ts:70]: What's the difference between adding it to stack directly and adding it to `ITemplateOption` [here](https://github.com/aws/aws-cdk/blob/v2.44.0/packages/@aws-cdk/core/lib/stack.ts#L1269)?

Comment (otaviomacedo) [reply src/evaluate/evaluate.ts:70]: Same thing, but I don't have to worry whether the array has been initialized:

https://github.com/aws/aws-cdk/blob/0358d51954b3ae32c6d7bdb490d498ab743770ec/packages/%40aws-cdk/core/lib/stack.ts#L726-L731

**Review by lejiati — APPROVED** [review]

**Comment by otaviomacedo** [comment]
> I assume this change is only to support transforms declared at template level?

Yes. What else are you thinking about? The [`Fn::Transform`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-transform.html) intrinsic function?

**Comment by lejiati** [comment]
> Yes. What else are you thinking about? The [Fn::Transform](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-transform.html) intrinsic function?

Yea I think the issue was meant to cover that as well. But we can have another issue to track that.

**Pull request merged**